packages/sitecore-jss-react/src/components/BYOCComponent.tsx

```diff
@@ -175,31 +174,12 @@ export async function fetchBYOCComponentServerProps(
     : {};
 
   try {
-    fetchedData = await fetchData(fetchDataOptions);
+    fetchedData = await FEAAS.DataSettings.fetch(fetchDataOptions || {});
   } catch (e) {
-    console.error(e);
+    console.error('Fetch BYOC component props failed');
```

Comment:
Do we still want to throw an error? As we do for "FEaaSComponent"

Reply:
Yeah you're right it by default returns an empty object. No use of throwing an error.

Reply:
@sc-addypathania Nope, I mean in case there is an error we need to throw it, currently it's supressed

Reply:
so the `FEAAS.DataSettings.fetch` doesn't throw any error in case of error it return `{}`
So above in the component i have added check for if the `fetchedData` is null.

Reply:
@sc-addypathania Makes sense, if it throws '{}' in any case, it's fine